docs/multicluster/architecture.md

```diff
@@ -57,7 +57,7 @@ and encapsulates them into ResourceImports.
 
 ## Service Export and Import
 
-<img src="assets/resource-export-import-pipeline.png" width="800" alt="Antrea Multi-cluster Resource Export/Import Pipeline">
+<img src="assets/resource-export-import-pipeline.svg" width="800" alt="Antrea Multi-cluster Resource Export/Import Pipeline">
```

Comment:
The picture is a little small too (hard for me to read the texts).

Reply:
In the picture, consider:

1. Endpoint -> Endpoints
2. Monitor -> Watch?

Reply:
Done